docs/man_pages/index.md

```diff
@@ -41,6 +41,7 @@ Command | Description
 [mobileframework](project/configuration/mobileframework.html) | Lists all supported versions of the current development framework.
 [mobileframework&nbsp;set](project/configuration/mobileframework-set.html) | Sets the selected development framework version for the project.
 [update-kendoui](lib-management/update-kendoui.html) | Updates or adds Kendo UI Core or Kendo UI Professional to your project.
+[generate](generate.html) | Generates splashscreens or icons based on a single high-resolution image and saves them to the project.
```

Comment:
Personally, I find that "generate" is a too generic command name that might mean anything. Are we planning on expanding this to generate even more resources?

For example, ionic are using resources as the command and --icon and --splash as the options. Further, we could use resources without any options to list the current resources and their locations.